src/test/java/com/epam/tc/hw7/ex1/TestData.java

```diff
@@ -0,0 +1,94 @@
+package com.epam.tc.hw7.ex1;
+
+import com.fasterxml.jackson.core.JsonFactory;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonToken;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import lombok.SneakyThrows;
+import org.apache.commons.lang3.StringUtils;
+import org.testng.annotations.DataProvider;
+
+public class TestData {
```

Comment:
What kind of test data is this? Name of the class should be created with reference to it purpose.